.github/workflows/github-actions-release-version.yml

```diff
@@ -38,6 +43,12 @@ jobs:
 
       - name: Build AAR ⚙️🛠
         run: bash ./gradlew :NeuroID:assemble
+        env:
+            GPR_USER: ${{ secrets.GPR_USER }}
+            GPR_API_KEY: ${{ secrets.GPR_API_KEY }}
+
```

Comment:
Do you need the token to build the packages? I was under the impression you needed them for uploading them?

Reply:
The build command kicks off the [build.gradle](https://github.com/Neuro-ID/neuroid-android-sdk/pull/152/files#diff-68e4b72acc46b2cf6775e24ad45d4bdc42bf72e02b44b96f5929bc88b14f95fb) file in NeuroID folder that sets these values to ready for publishing .